src/pylint_odoo/checkers/odoo_addons.py

```diff
@@ -1216,6 +1229,24 @@ def visit_functiondef(self, node):
                 there_is_super = True
                 break
 
+        # Verify if super attributes are prohibited methods to override
+        if there_is_super and (
+            self.linter.config.prohibited_method_override or DFTL_PROHIBITED_OVERRIDE_METHODS
```

Comment:
This line is causing one of the linters to fail (black). Could you fix it, please?

You just need to replace it by:

`if there_is_super and self.linter.config.prohibited_method_override or DFTL_PROHIBITED_OVERRIDE_METHODS:`